subt/octomap.py

```diff
@@ -350,6 +345,19 @@ def update(self):
                     paused = not paused
                 if ord('0') <= key <= ord('9'):
                     level = key - ord('0')
+                if args.open3d and key == ord('d'):
+                    import open3d as o3d
```

Comment:
Do you want this in the docker image so that it works for everybody?

The counterargument is docker image size and I am not sure if anybody except me is running even debugging tools inside the container.

Reply:
p.s. OTOH now I was wandering why the 3D view does not work?! ... because I forgot to add the parameter ... hmm ...

Reply:
I would maybe wait if `open3d` would be needed for normal processing. You can still install it in your docker (I have one open for a "very long time") until `Dockerfile` changes. So I would wait for another need to have it available.